modules/manage/partials/authentication.adoc

```diff
@@ -1847,7 +1847,7 @@ schema_registry:
     authentication_method: none
 ----
 
-If authorization is enabled, connections to this listener use the anonymous user.
+If authorization is disabled, connections to this listener use the anonymous user.
```

Comment:
this is a section about disabling auth, and it doesn't make sense to use anonymous user if auth is enabled.